CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ set(DEPS_IGNORE_SHA FALSE)
 #-------------------------------------------------------------------------------
 set(FUNCS_DATA ${PROJECT_BINARY_DIR}/funcs_data.mpack)
 set(TOUCHES_DIR ${PROJECT_BINARY_DIR}/touches)
+set(VTERM_TEST_FILE ${PROJECT_BINARY_DIR}/test/vterm_test_output)
```

Comment:
if this is a test artifact, should this go in the build/ dir? that is where we store e.g. tempfiles created by `t.tmpname()`

Reply:
`PROJECT_BINARY_DIR` is the `build/` dir.